config/SOFTWARE-LIST.tpl.md

```diff
@@ -0,0 +1,20 @@
+# CISA Log4j (CVE-2021-44228) Affected Vendor & Software List #
+
+## Status Descriptions ##
+
+| Status | Description |
+| ------ | ----------- |
+| Unknown | Status unknown. Default choice. |
+| Affected | Reported to be affected by CVE-2021-44228. |
+| Not Affected | Reported to NOT be affected by CVE-2021-44228 and no further action necessary. |
+| Fixed | Patch and/or mitigations available (see provided links). |
+| Under Investigation | Vendor investigating status. |
+
+## Software List ##
+
+This list was initially populated using information from the following sources:
+
+- Kevin Beaumont
+- SwitHak
+
+{{software_markdown_table}}
```

Comment:
I think we should mention something like this:
```suggestion
NOTE: This file is automatically generated.  To make updates, please refer to
[`CONTRIBUTING.md`](CONTRIBUTING.md).

{{software_markdown_table}}
```

Reply:
@dav3r Good addition!

Reply:
Added in 3991f5e.